components/ThemeWrapper.test.js

```diff
@@ -16,73 +20,29 @@ describe(ThemeWrapper.name, () => {
 
   it('should use custom storage key', () => {
     const key = 'my-custom-key'
-    const results = render(ThemeWrapper, {
-      props: {
-        key,
-      },
-    })
-    expect(localStorage.getItem(key)).toBeTruthy()
+    const results = TestHarness({ key })
     expect(localStorage.getItem(key)).toEqual(presets[0].name)
   })
 
   it('should throw error if supplied with empty themes', () => {
-    const themes = []
-    try {
-      const results = render(ThemeWrapper, {
-        props: {
-          themes,
-        },
-      })
-    } catch (error) {
-      expect(error.message).toEqual(INVALID_THEMES_MESSAGE)
-    }
+    expect(() => TestHarness({ themes: [] })).toThrow(INVALID_THEMES_MESSAGE)
```

Comment:
You can trap exceptions by wrapping the method with an anonymous function in your expect() statement.

Example using previous code 
```js
expect(() => render(ThemeWrapper, { props: { themes } })).toThrow(INVALID_THEMES_MESSAGE)
```

Reply:
TIL! This is the first Svelte project I added tests to, I'm still pretty new to it :)